runtime/autoload/provider/pythonx.vim

```diff
@@ -13,19 +13,21 @@ function! provider#pythonx#Detect(ver) abort
   let skip = exists(skip_var) ? {skip_var} : 0
   if exists(host_var)
     " Disable auto detection
-    let [check, err] = s:check_interpreter({host_var}, a:ver, skip)
+    let [check, err, _] = s:check_interpreter({host_var}, a:ver, skip)
     return check ? [{host_var}, err] : ['', err]
   endif
 
   let detect_versions = (a:ver == 2) ?
-        \   ['2.7', '2.6', '2', '']
-        \ : ['3.5', '3.4', '3.3', '3.2', '3', '']
+        \   ['2', '2.7', '2.6', '']
```

Comment:
> In my case as long as you test 2.6 and 2.7 before 'python' it would work (or in the worst case I can make it work without too much trouble and maintenance). Thanks for asking 

@jonthn It would work even if 'python' is tested before 2.7, as only 2.7 would have installed the `neovim` module, right?